app/views/scenarios/_form.html.erb

```diff
@@ -1,3 +1,16 @@
+<script type="text/javascript">
+function format(icon) {
+  var originalOption = icon.element;
+  return '<i class="fa ' + $(originalOption).data('icon') + '"></i> ' + icon.text;
+}
+$(document).ready(function() {
+  $(".js-example-basic-single").select2({
+    width: '100%',
+    formatResult: format
+  });
+});
+</script>
+
```

Comment:
I can convert this into coffee script, should I put the later converted version in application.js.coffee. There is currently no coffee page associated with scenario.